src/pages/document/[id].tsx

```diff
@@ -307,13 +350,21 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   }
 
   if (familyData) {
+    geographyCode = familyData.geography;
     try {
       const targetsRaw = await axios.get<TTarget[]>(`${process.env.S3_PATH}/families/${familyData.import_id}.json`);
       targetsData = targetsRaw.data;
     } catch (error) {}
   }
 
-  if (!familyData) {
+  if (geographyCode) {
+    try {
+      const { data: returnedData }: { data: TGeographySummary } = await client.get(`/summaries/geography/${geographyCode}`);
+      geographySummaryData = returnedData;
+    } catch (error) {}
+  }
+
+  if (!familyData || !geographySummaryData) {
```

Comment:
I personally don't think we should prevent the entire page loading if we don't get the geo summary data loaded.

Similar to how we handle targets, just hide the search box / don't display the country-specific data if we don't have it.